### PR TITLE
Less than equal bloqs.

### PR DESCRIPTION
**Pull request by fdmalone**
Wrap some of cirqs comparators.

### CODE REVIEW

File: qualtran/bloqs/arithmetic.py [hunk 1]
```diff
@@ -212,3 +219,74 @@ def t_complexity(self):
         # See: https://github.com/quantumlib/cirq-qubitization/issues/219
         # See: https://github.com/quantumlib/cirq-qubitization/issues/217
         return TComplexity(t=8 * self.bitsize)
+
+
+@frozen
+class LessThanEqual(Bloq):
+    r"""Implements $U|x,y,z\rangle = |x, y, x \le y\rangle$.
```
Comment (mpharrigan) [inline qualtran/bloqs/arithmetic.py:226]: z has dissapeared in your telling. Either change to |x, y, 0> -> x, y, x<=y or a z \oplus

File: qualtran/bloqs/arithmetic.py [hunk 2]
```diff
@@ -212,3 +219,74 @@ def t_complexity(self):
...
+
+    def decompose_bloq(self) -> 'CompositeBloq':
+        return decompose_from_cirq_op(self)
+
+    def as_cirq_op(
+        self, qubit_manager: 'cirq.QubitManager', **cirq_quregs: 'CirqQuregT'
+    ) -> Tuple[Union['cirq.Operation', None], Dict[str, 'CirqQuregT']]:
+        less_than = CirqLessThanEqual(x_bitsize=self.x_bitsize, y_bitsize=self.y_bitsize)
+        x = cirq_quregs['x'].tolist()
+        y = cirq_quregs['y'].tolist()
```
Comment (mpharrigan) [inline qualtran/bloqs/arithmetic.py:258]: is the tolist necessary? can't you `*args` unpack numpy arrays as well? please write a small comment if it is indeed required

File: qualtran/bloqs/arithmetic.py [hunk 3]
```diff
@@ -212,3 +219,74 @@ def t_complexity(self):
...
+        return Signature.build(x=self.bitsize, z=1)
+
+    def decompose_bloq(self) -> 'CompositeBloq':
+        return decompose_from_cirq_op(self)
+
+    def as_cirq_op(
+        self, qubit_manager: 'cirq.QubitManager', **cirq_quregs: 'CirqQuregT'
+    ) -> Tuple[Union['cirq.Operation', None], Dict[str, 'CirqQuregT']]:
+        less_than = CirqLessThanGate(bitsize=self.bitsize, less_than_val=self.val)
+        x = cirq_quregs['x'].tolist()
```
Comment (mpharrigan) [inline qualtran/bloqs/arithmetic.py:290]: ditto

File: qualtran/bloqs/arithmetic_test.py [hunk 4]
```diff
@@ -96,5 +105,16 @@ def test_greater_than():
     cbloq = bb.finalize(a=q0, b=q1, result=anc)
 
 
+def test_less_than_equal():
+    lte = LessThanEqual(5, 5)
+    print(lte.decompose_bloq().t_complexity())
```
Comment (mpharrigan) [inline qualtran/bloqs/arithmetic_test.py:110]: remove print. can this be turned into an assertion?

File: qualtran/bloqs/arithmetic_test.py [hunk 5]
```diff
@@ -96,5 +105,16 @@ def test_greater_than():
...
+    qlt_testing.assert_valid_bloq_decomposition(lte)
+
+
+def test_less_than_constant():
+    lte = LessThanConstant(5, 7)
```
Comment (mpharrigan) [inline qualtran/bloqs/arithmetic_test.py:115]: lte -> ltc

File: qualtran/bloqs/arithmetic_test.py (hunk 4 above)
Comment (fdmalone) [reply qualtran/bloqs/arithmetic_test.py:110]: whoops

**Pull request merged**